server/lib/topStories.js

```diff
@@ -1,29 +1,39 @@
+/* eslint strict:0 */
 const fetch = require('node-fetch');
 const briefArticle = require('./viewModel');
 const R = require('ramda');
 const briefArticleMapper = R.map(briefArticle);
 
 const siteApiEndpoint = process.env.siteApiEndpoint;
-const frontPageId = process.env.frontPageId;
 const mainContentKey = process.env.mainContentKey;
 const apiKey = process.env.apiKey;
+const frontPageIdUK = process.env.frontPageId;
+const frontPageIdUS = process.env.frontPageIdUS;
```

Comment:
yes, good point. the only id that needs to be secret is the apikey.
Can't see where in history the pageId was made explicit.
Will make the change.